src/executables/util_cs.c

```diff
@@ -3994,6 +3994,798 @@ tde (UTIL_FUNCTION_ARG * arg)
   return EXIT_FAILURE;
 }
 
+static int
+get_class_index (int num_class, OID * oidlist, OID classoid)
+{
+  int idx = 0;
+
+  for (idx = 0; idx < num_class; idx++)
+    {
+      if (OID_EQ (&oidlist[idx], &classoid))
+	{
+	  return idx;
+	}
+    }
+
+  return -1;
+}
+
+static int
+check_sql_memory (char **sql, int req_size, int *max_sql_size)
```

Comment:
It does not only check but also resize. The name should include this work.

Reply:
yes, I agree with you